### PR TITLE
Only report captions when entering and exiting them

### LINKED ISSUE #14874: NVDA announces 'caption' before every element within figcaption element
### Steps to reproduce:
Author a webpage containing a figure and figcaption element. Add links to the figcaption element (as allowed in the HTML spec)
e.g. https://codepen.io/clanceyp/full/RweVbBj

### Actual behavior:
NVDA announces "caption" multiple times through out the figcaption content
e.g. NVDA speaks "caption link [link label]" for each link element within a figcaption

Screenshot 2023-04-25 at 19 02 02

### Expected behavior:
NVDA announces "caption" a single time at the beginning of the figcaption content
e.g. NVDA speaks "link [link label]" for each link element within a figcaption

### NVDA logs, crash dumps and other attachments:
NA
### System configuration
NA
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2023.1
#### Windows version:
10
#### Name and version of other software in use when reproducing the issue:
MS Edge 112.0.1722.58
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No have not tried.
#### If NVDA add-ons are disabled, is your problem still occurring?
no add-ons installed
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

### PR DESCRIPTION
**Pull request by SaschaCowley**
### Link to issue number:

Fixes #14874 

### Summary of the issue:

NVDA reports the word "caption" before every element in a `figcaption` element, or before each line in same when navigating by line in browse mode.

### Description of user facing changes

NVDA now reports "caption" only when first entering a figure caption, and "out of caption" when leaving it.

### Description of development approach

Changed `textInfos.ControlField.getPresentationCategory` to return `PRESCAT_CONTAINER` when its role is `controlTypes.Role.CAPTION` .

### Testing strategy:

Manually tested in Firefox with my [figure example webpage](https://saschacowley.com/demos/figure)

### Known issues with pull request:

None

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [ ] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

**Comment by seanbudd** [comment]
is this ready for review?

**Comment by SaschaCowley** [comment]
> is this ready for review?

Apologies, I was waiting for checks to complete last week, and completely forgot to mark it as ready!

**Review by gerald-hartig — APPROVED** [review]

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -60,6 +60,7 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
 - In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
 - NVDA will keep the audio device awake after speech stops, in order to prevent the start of the next speech being clipped with some audio devices such as Bluetooth headphones. (#14386, @jcsteh, @mltony)
 - HP Secure Browser is now supported. (#16377)
+- When reading by line in browse mode, "caption" is no-longer reported on each line of a long figure or table caption. (#14874)
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:63]:
```suggestion
- When reading by line in browse mode, "caption" is no longer reported on each line of a long figure or table caption. (#14874)
```

**Review by seanbudd — APPROVED** [review]

**Comment by seanbudd** [comment]
Could you move the changes to the new changes.md for this PR please?

**Pull request merged**